.github/workflows/integration-test.yml

```diff
@@ -39,22 +41,25 @@ jobs:
 
       - name: Upload debug transaction logs
         if: always()
-        uses: actions/upload-artifact@v2
+        # https://github.com/actions/upload-artifact/releases/tag/v2.3.1
+        uses: actions/upload-artifact@82c141cc518b40d92cc801eee768e7aafc9c2fa2
         with:
           name: test-transacion-logs
           path: tmp
 
       - name: Collect docker logs on failure
         if: failure()
-        uses: jwalton/gh-docker-logs@v1
+        # https://github.com/jwalton/gh-docker-logs/releases/tag/v1.0.0
```

Comment:
Maybe we should take the opportunity to upgrade this version to the latest (https://github.com/jwalton/gh-docker-logs/releases/tag/v2.2.1). v1.0.0 is a very old version.

Apparently their only breaking change was to use `/bin/sh` instead of `/bin/bash`, but this is configurable, so if we have problems with this we would just have to explicitly set to `/bin/bash` again. (see https://github.com/jwalton/gh-docker-logs/releases/tag/v2.0.0)

Reply:
That's a good idea! I believe these updates will be better left as a separate PR, and that could also be implemented in our other repositories as well.